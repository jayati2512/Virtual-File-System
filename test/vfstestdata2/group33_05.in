createvfs vfs_01 1024
createvfs vfs_01 1024
createvfs vfs_02 2000
createvfs vfs_03 -1
createvfs vfs/1 1024
createvfs anbfghtchtendtjkijswotwejsqrtytoonfjgnghd 1024
createvfs 10
mountvfs vfs
mountvfs
mountvfs vfs_01
unmountvfs
unmountvfs vfs_01
unmountvfs vfs_01
