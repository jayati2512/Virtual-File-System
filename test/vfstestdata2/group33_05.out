createvfs_SUCCESS
createvfs_FAILURE DATA_FILE_ALREADY_EXISTS
createvfs_FAILURE INVALID_SIZE
createvfs_FAILURE INVALID_SIZE
createvfs_FAILURE INVALID_CHARACTER_IN_NAME
createvfs_FAILURE VFS_LABEL_TOO_LARGE
createvfs_FAILURE VFS_INSUFFICIENT_ARGUMENTS
mountvfs_FAILURE DATA_FILE_NOT_FOUND
mountvfs_FAILURE VFS_INSUFFICIENT_ARGUMENTS
mountvfs_SUCCESS
unmountvfs_FAILURE VFS_INSUFFICIENT_ARGUMENTS
unmountvfs_SUCCESS
unmountvfs_FAILURE VFS_NOT_MOUNTED
